### ISSUE TITLE
Migrate ASAP!

### ISSUE THREAD

**Issue by NotNotMarshall** [jsibbold/zoomage#110]
jCenter is currently deprecated and artifacts can only be downloaded till May of 2021, consider migrating to jitpack or maven central. Your library is already breaking.
more info here:
https://developer.android.com/studio/build/jcenter-migration

**Comment by aberaud** [comment]
My project build is failing because of this:
```
22:47:58    > Could not resolve com.jsibbold:zoomage:1.2.0.
22:47:58      Required by:
22:47:58          project :app
22:47:58       > Could not resolve com.jsibbold:zoomage:1.2.0.
22:47:58          > Could not get resource 'https://jcenter.bintray.com/com/jsibbold/zoomage/1.2.0/zoomage-1.2.0.pom'.
22:47:58             > Could not GET 'https://jcenter.bintray.com/com/jsibbold/zoomage/1.2.0/zoomage-1.2.0.pom'. Received status code 403 from server: Forbidden
```

If not fixed ASAP I'm gonna have to remove this library from my project :(

**Comment by jsibbold** [comment]
working on finding it a new home

**Comment by aberaud** [comment]
Indeed, this was another issue and the file will remain available on jcenter for a while.

**Comment by NotNotMarshall** [comment]
> Working on finding it a new home, though I believe they have shifted the deadline to February 2022. Current packages should be available until that time.

Good luck and thanks for your hard work!

**Comment by skovacevic96** [comment]
Any news regarding this? It is really not necessary to leave jCenter in my project just because of this library

**Comment by NotNotMarshall** [comment]
> Any news regarding this? It is really not necessary to leave jCenter in my project just because of this library

I have some good news on this, they have decided to not delete the projects from JCenter but rather make them Read-Only, but no further updates or writes will be possible. New versions of libraries must choose another place to host.

**Comment by jsibbold** [comment]
Yeah, the plan is still to move it to maven central, but yes, thankfully, current versions will remain available.

**Comment by developernotes** [comment]
Hi @jsibbold 

Any updates on the plan to move zoomage to Maven Central?

**Comment by skovacevic96** [comment]
Hi, @jsibbold any news? We could help you out if you don't have much time.

**Comment by jsibbold** [comment]
Still here, still working on it. It's been a rough year. I'll try not to leave you hanging too much longer.

**Comment by DanielGalarza** [comment]
> v1.3.1 should now be available on Maven Central

Would you mind updating your README with the new way to pull this dependency, please?

**Comment by jsibbold** [comment]
README updated, thanks for reminding me
